### ISSUE TITLE
Whitespace in new view of cell

### ISSUE THREAD

**Issue by OliverEvans96** [jupyterlab/jupyterlab#3228]
Hi all,

Thanks for all the hard work put into redoing mirroring output the correct way in #3159. I agree that it makes more sense to include the whole cell rather than individual outputs, which may not make sense out of context.

One downside is the whitespace which comes with the layout of a full cell. In trying to fit all of the interaction possible into one screen, it's important not to waste any real estate. It seems a bit unnecessary to have a one-inch column of white space right in the middle of the screen.
![wasted_pixels](https://user-images.githubusercontent.com/15915043/32686048-eea25da0-c66b-11e7-99f6-a8feff2745a3.png)

I understand that having the execution number can be an important part of the context, but I'm wondering if it would be possible to minimize the whitespace, or if anyone else might have thoughts about how to improve the layout.

Thanks again for the great work!
Oliver

**Comment by jasongrout** [comment]
Definitely. It's nice to have the cell label up at the top, so it's easy to match up to the full notebook, but I agree that in this case, the wasted space is not worth it in its current form. It would be great to have a UI that gives that prompt number without wasting all that space along the side.

**Comment by jasongrout** [comment]
Do you want to take a look at what is needed to get rid of that gutter? It may involve changes to the cell (i.e., adding an option to show the prompt or something). The changes are probably in the cell classes that were changed in #3159.
